app/src/layouts/DashboardLayout/DashboardContent.js

```diff
@@ -191,6 +194,11 @@ const DashboardContent = () => {
             )}
 
           {/* {isJoinWorkspaceCardVisible && user.loggedIn ? <JoinWorkspaceCard /> : null} */}
+          {appMode === GLOBAL_CONSTANTS.APP_MODES.DESKTOP ||
+          isWorkspaceOnboardingCompleted ||
+          appOnboardingDetails.isOnboardingCompleted ? (
+            <RequestBillingTeamAccessModal isReminder />
```

Comment:
this could be a separate component entirely, since we are creating a new copy and also introducing a lot of specific renderring logic for this case that overrides almost everything inside this component.